src/WpBookingsUiModule.php

```diff
@@ -123,18 +119,11 @@ public function run(ContainerInterface $c = null)
             $this->_adminMenu($c);
         });
 
-        $this->_attach('eddbk_bookings_visible_statuses', function ($event) use ($c) {
-            $event->setParams([
-                'statuses' => $this->_getVisibleStatuses($event->getParam('statuses'), $c->get('wp_bookings_ui/hidden_statuses')),
-            ]);
-        });
+        $this->_attach('eddbk_bookings_ui_state', $c->get('eddbk_bookings_ui_statuses_handler'));
```

Comment:
What happens if somewhere else we need a map of status codes to translated status labels? What happens when we need to modify the list of booking statuses that are visible to the user - but in response to a different request?